scripts/install-prometheus-operator.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -o errexit
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+# Apply the crds and wait for them
+git clone https://github.com/prometheus-operator/kube-prometheus.git --depth 1 -b v0.11.0
+cd kube-prometheus
+oc apply --server-side -f manifests/setup
+oc wait \
+	--for condition=Established \
+	--all CustomResourceDefinition \
+	--namespace=monitoring
+oc apply -f manifests/
+oc wait \
```

Comment:
I would add the `--timeout="$TNF_DEPLOYMENT_TIMEOUT"` option here. Otherwise, it will probably use [30secs ](https://github.com/openshift/oc/blob/d14fc102c6d2806268c3ce2e11877d23d8b7223a/vendor/k8s.io/kubectl/pkg/cmd/wait/wait.go#L109) as default timeout. It might be enough for the previous `oc wait` (line 10), as it's waiting for CRDs to be deployed, but I think it could be a bit low for pods to deploy on time.

I tested this PR locally in my kind cluster and got a timeout:
```
servicemonitor.monitoring.coreos.com/prometheus-operator created
+ oc wait --for condition=available --all deployments --namespace=monitoring
deployment.apps/grafana condition met
deployment.apps/kube-state-metrics condition met
deployment.apps/prometheus-adapter condition met
deployment.apps/prometheus-operator condition met
error: timed out waiting for the condition on deployments/blackbox-exporter
make: *** [Makefile:23: install] Error 1
```

Reply:
You added the timeout to the wrong wait :-) Add the timeout to both waits. Also add default in case the variable is not set:
```
[ -z "$TNF_DEPLOYMENT_TIMEOUT" ] || TNF_DEPLOYMENT_TIMEOUT=30
```

Reply:
Yes, she added it in the wrong `oc wait`. By the way, @rdavid , the timeout env var is always exported in the `init-env.sh` script:
```
export TNF_DEPLOYMENT_TIMEOUT="${TNF_DEPLOYMENT_TIMEOUT:-240s}"
```
Do you really think that check is necessary here?

Reply:
@greyerof @shimritproj in this case, no need for additional default. Maybe add a comment that the variable is set in `init-env.sh`.

By the way, the expansion in `init-env.sh` is more elegant way to handle default than mine.